Chinese/Demo_htx/demo.md

```diff
@@ -0,0 +1,2 @@
+click [here](http://www.baidu.com) to visit baidu
+click [here](https://github.com/udacity/create-your-own-adventure) to visit raw url
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.